### PR TITLE
Tempo - Update the main external link to distributed tracing

### LINKED ISSUE #6687: Tempo - Update the main external link to distributed tracing
Tempo - Update the main external link to distributed tracing, in Tempo it redirects to the API url, that returns a 404 as it not serve requests in /. 

image

### PR DESCRIPTION
**Pull request by josunect**
Fixes #6687 

image

Not sure if we should hide the query param in the url link?

### CODE REVIEW

File: frontend/src/components/About/AboutUIModal.tsx [hunk 1]
```diff
@@ -68,6 +69,18 @@ export const AboutUIModal: React.FC<AboutUIModalProps> = (props: AboutUIModalPro
     );
   };
 
+  const renderTempo = (externalServices: ExternalServiceInfo[]) => {
+    const tempoService = externalServices.find(service => service.name === 'tempo');
```
Comment (hhovsepy) [inline frontend/src/components/About/AboutUIModal.tsx:73]:
```suggestion
    const tempoService = externalServices.find(service => service.name === TEMPO);
```

Comment (josunect) [reply frontend/src/components/About/AboutUIModal.tsx]: Thanks! Committed suggestion.

**Review by hhovsepy — APPROVED** [review]
Lgtm, tested locally:
Screenshot from 2023-10-18 11-58-19

**Comment by josunect** [comment]
> Lgtm, tested locally: Screenshot from 2023-10-18 11-58-19

Thank you, @hhovsepy !

**Pull request merged**